rest/client/testdata/dynamic_idle_timeout_task.yaml

```diff
@@ -21,7 +21,7 @@ tasks:
     commands:
       - command: timeout.update
         params:
-          timeout_secs: 2
+          timeout_secs: ${my_new_timeout}
```

Comment:
test passing both an expansion and an actual number?

Reply:
The other test does this.